prog/vm/github_runner.rb

```diff
@@ -224,7 +224,7 @@ def setup_info
     COMMAND
 
     if github_runner.installation.cache_enabled
-      local_ip = vm.nics.first.private_ipv4.network.to_s
+      local_ip = vm.private_ipv4.to_s
       command += <<~COMMAND
         echo "CUSTOM_ACTIONS_CACHE_URL=http://#{local_ip}:51123/random_token/" | sudo tee -a /etc/environment
```

Comment:
nit: no need to `to_s`

```suggestion
      command += <<~COMMAND
        echo "CUSTOM_ACTIONS_CACHE_URL=http://#{vm.private_ipv4}:51123/random_token/" | sudo tee -a /etc/environment
```